config/level1/Boss.py3

```diff
@@ -100,14 +101,14 @@ class Tile:
 # game loop
 while True:
     # Here are my local variables, these are reset at each turn
-    playerinfos = []
+    myHeroPos = Vector(0,0)
     quests = []
     items = []
 
     board = {}  # The board is a dictionnary linking the position of a tile to the tile object
 
     # Here we get the type of turn : 0 for push and 1 for move
-    turn_type = int(input())
+    turnType = int(input())
```

Comment:
In Python, we use lower case with underscores for variables and functions:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
Variable names follow the same convention as function names.

However, it's not a real issue and it's better to have one style that a mix between snake case and camel case :+1:.

Reply:
Resolve this?

Reply:
Yes.